src/bioregistry/data/bioregistry.json

```diff
@@ -54703,6 +54683,257 @@
       }
     }
   },
+  "iedb.assay": {
+    "contributor": {
+      "email": "m.naguthana@hotmail.com",
+      "github": "nagutm",
+      "name": "Mufaddal Naguthanawala",
+      "orcid": "0009-0009-5240-7463"
+    },
+    "description": "Identifiers for specific experimental assays that serve as evidence for immune-related studies.",
+    "example": "1691753",
+    "github_request_issue": 1204,
+    "homepage": "https://www.iedb.org/",
+    "name": "Immune Epitope Database",
+    "part_of": "iedb",
+    "pattern": "^\\d+$",
+    "publications": [
+      {
+        "doi": "10.1093/nar/gku938",
+        "pmc": "PMC4384014",
+        "pubmed": "25300482",
+        "title": "The immune epitope database (IEDB) 3.0",
+        "year": 2014
+      },
+      {
+        "doi": "10.1111/j.1365-2567.2012.03611.x",
+        "pmc": "PMC3461392",
+        "pubmed": "22681406",
+        "title": "The immune epitope database: a historical retrospective of the first decade",
+        "year": 2012
+      },
+      {
+        "doi": "10.1093/nar/gkp1004",
+        "pmc": "PMC2808938",
+        "pubmed": "19906713",
+        "title": "The immune epitope database 2.0",
+        "year": 2009
+      },
+      {
+        "pubmed": "16312048",
+        "title": "The immune epitope database and analysis resource: from vision to blueprint",
+        "year": 2004
+      }
+    ],
+    "uri_format": "https://www.iedb.org/assay/$1"
+  },
+  "iedb.epitope": {
```

Comment:
this record needs to retain the miriam link

Reply:
The identifiers.org record appears to be specific to `iedb.reference`, could you elaborate why `iedb.epitope` should retain that link?

Reply:
might be that the diff is too hard to read, it looked like the URI pattern from the MIRIAM record matched the `iedb.epitope` entry. Will take another look later